self-samples/src/App.js

```diff
@@ -2,11 +2,13 @@ import React from 'react';
 import './App.css';
 
 import Sample from './components/Sample';
+import Krishna from './components/Krishna';
```

Comment:
Please revert change in this file. It would be difficult for others to resolve merge conflicts. Other file is fine

Reply:
Please review